app.js

```diff
@@ -0,0 +1,74 @@
+var arr = 
+[
+  {
+    name: "Lion King",
+    license: [2011, 2013]
+  },
+
+  {
+    name: "Iron Man",
+    license: [2005,2010]
+  },
+  {
+    name: "Life of Pi",
+    license: [2008,2014]
+  },
+  {
+    name: "Another Movie",
+    license: [2011,2013]
+  }
+];
```

Comment:
Let's put this in a separate file

Also, let's use `const`

Reply:
Using const. How do I import from a separate file ?